src/status_im/contexts/preview/quo/list_items/network_list_item.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im.contexts.preview.quo.list-items.network-list-item
```

Comment:
let's match the figma name here `status-im.contexts.preview.quo.list-items.network-list`

Reply:
👍